content/docs/user-guide/experiment-management/running-experiments.md

```diff
@@ -20,7 +20,7 @@ experiment(s). These files codify _pipelines_ that specify one or more
 
 ### Running the pipeline(s)
 
-You can run the experimental pipeline using `dvc exp run`. It uses `./dvc.yaml`
+You can run the experiment pipeline using `dvc exp run`. It uses `./dvc.yaml`
```

Comment:
```suggestion
You can run the pipeline of the experiment with `dvc exp run`. It uses `./dvc.yaml`
```

Relates to a point in https://github.com/iterative/dvc.org/issues/2768:

> removing the pipeline and stages concept would be ideal

Reply:
For now, I'm not sure what's wrong with "experimental pipeline" though. Isn't that what it is?

Reply:
Hmm, the way I read it -  "experimental pipeline" suggests not only possession ("of experiment") but that the pipeline itself is somehow of experimental nature (i.e. not final?).
Here, we just want to convey possession to the experiment. There's nothing "experimental" about the pipeline *itself*.

Reply:
I think "experiment pipeline" is better than "pipeline of the experiment"

Reply:
"experimental" is not possessive though 🙂  "experiment's" would be possessive. "Experimental" is a qualitative adjective: it indicates a quality of the noun ("pipeline" here). We do not want to express possession above, we're just talking about the pipeline (which is itself an experiment).

> that the pipeline itself is somehow of experimental nature (i.e. not final?)

Yes, that's the intended meaning of the phrase. The hypothetical pipeline here is experimental in fact.

The problem IMO is the unnecessary complication of having the pipeline concept here, which again is something we want to address per #2768.

Reply:
> something we want to address separately, per https://github.com/iterative/dvc.org/issues/2768

Now that if you want to address that here, why not?

Reply:
p.s. I also have no strong opinion agains "pipeline of the experiment" TBH, esp. since we want to remove the pipeline concept later so this will be reviewed again then.

Reply:
> "Yes, that's the intended meaning of the phrase. The hypothetical pipeline here is experimental in fact."

Have to admit I'm a bit confused about that sentence 😄 
@jorgeorpinel  - I don't want to go against your judgment & expertise here 
IMO - `experiment pipeline` is more appropriate here than `experimental pipeline`, but definitely can live with both.
I'll go with whichever you decide

Reply:
K let's keep your change 👍🏼

Reply:
🙏